letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -287,6 +287,10 @@ BootstrapRpmCommon() {
 
   if ! $SUDO $tool list *virtualenv >/dev/null 2>&1; then
     echo "To use Certbot, packages from the EPEL repository need to be installed."
+    if ! $SUDO $tool list epel-release >/dev/null 2>&1; then
```

Comment:
Actually I guess this is fine, because we're inside the `if` about virtualenv. So overall, LGTM.